### DISCUSSION TITLE
Kindly include doc's link

### DISCUSSION THREAD

**Discussion by surapuramakhil** [sourcenetwork/defradb#3319] [Feature request]
### Is the request related to a problem?

initilaly felt this project doesn't have any docs 

but after research found this https://docs.source.network/defradb

### The feature you would like

Humble request to attach this link to repo so that users can quickly find docs

**Comment by fredcarle** [comment]
Hi @surapuramakhil, the link to the docs website is in the top section of the Readme. When you say attach the link to the repo, does that mean something other than having the link in the readme?

**Reply by surapuramakhil** [reply]
yes, you can attach the link to the repo on the right side (under repo description)

something like this
<img width="1288" alt="Screenshot 2024-12-11 at 5 20 14 PM" src="https://github.com/user-attachments/assets/9e2487ce-2890-4123-8453-b92f8e02e340" />

**Reply by fredcarle** [reply]
The link had been added 🙂
